tests/Oryx.Integration.Tests/DotNetCore/DotNetCoreRuntimeVersion31Tests.cs

```diff
@@ -107,5 +107,50 @@ public async Task CanBuildAndRunApp_FromNestedOutputDirectory()
                     Assert.Contains("Welcome to ASP.NET Core MVC!", data);
                 });
         }
+
+        [Fact]
+        public async Task CanRunAppWhichUsesGDILibrary()
+        {
+            // Arrange
+            var appName = "ImageResizingWebApp";
+            var version = "3.1";
```

Comment:
I added this integration test only for 3.1 version. Feels like an overkill to have one for every runtime version even though ideally we should have one but probably I can add just runtime image tests checking for presence of `libgdiplus` library